pkg/admission/pluginconfig_webhook.go

```diff
@@ -123,6 +128,12 @@ func ValidateUpdatePluginConfig(ctx context.Context, c client.Client, old, obj r
 	); err != nil {
 		return nil, err
 	}
+	// Release migration is currently not supported. Thus, the ReleaseNamespace is immutable.
+	if err := validateImmutableField(oldPluginConfig.Spec.ReleaseNamespace, pluginConfig.Spec.ReleaseNamespace,
```

Comment:
```suggestion
	if err := validateImmutableField(oldPluginConfig.GetReleaseNamespace(), pluginConfig.GetReleaseNamespace(),
```

Reply:
True. That should fix the [mentioned contradiction](https://github.com/cloudoperators/greenhouse/pull/98#discussion_r15550505069) during migration